cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -698,6 +706,32 @@ private void createCluster(VertxTestContext context, Kafka kafka, List<Secret> s
                 // expected Secrets with certificates
                 assertThat(new TreeSet<>(secretsMap.keySet()), is(new TreeSet<>(expectedSecrets)));
 
+                // expected secret metrics emitted
+                if (metrics) {
```

Comment:
I think they are emitted regardless of this field, or? I think this has no impact on operator metrics.